include/swift/AST/DiagnosticsSema.def

```diff
@@ -2533,8 +2533,8 @@ NOTE(silence_inject_forced_downcast,none,
      "add parentheses around the cast to silence this warning", ())
 
 ERROR(conditional_downcast_foreign,none,
-      "conditional downcast to CoreFoundation type %0 will always succeed",
-      (Type))
+     "conditional downcast to CoreFoundation type %1 will always succeed; you would need to compare CFTypeID between %0 and %1",
```

Comment:
This ought to be a note (and phrased in the active voice).

Reply:
@CodaFi Thanks for looking into this. Do you mean that the suggest part should be in separated NOTE ?

Reply:
Yes

Reply:
Fixed! Please confirm.